extern/sector-storage/storiface/worker.go

```diff
@@ -77,33 +93,85 @@ var _ fmt.Stringer = &CallID{}
 	AddPiece(ctx context.Context, sector storage.SectorRef, pieceSizes []abi.UnpaddedPieceSize, newPieceSize abi.UnpaddedPieceSize, pieceData storage.Data) (CallID, error)
+
+	// SealPreCommit1 schedules a task to perform the `SealPreCommit1` stage of a Sector being sealed.
+	// If the worker does NOT have the Unsealed file for the given Sector, it will fetch it from a Worker that has it in order
+	// to execute the `SealPreCommit1`.
+	// The resulting Sealing files will be stored on the Worker on one of the Sealing scratch spaces of the sector store.
+	// The call will fail if the Worker is NOT able to allocate space to store the Unsealing & Sealing files.
 	SealPreCommit1(ctx context.Context, sector storage.SectorRef, ticket abi.SealRandomness, pieces []abi.PieceInfo) (CallID, error)
+
+	// SealPreCommit2 schedules a task to perform the `SealPreCommit2` stage of a sector being sealed.
+	// If the worker does NOT have the Sealing files for the Sector, it will fetch it from a worker that has them
+	// and store them on one of the Sealing scratch spaces of the sector store before executing the `SealPreCommit2`.
+	// The call will fail if the Worker is NOT able to allocate space to store the Sealing files.
 	SealPreCommit2(ctx context.Context, sector storage.SectorRef, pc1o storage.PreCommit1Out) (CallID, error)
+
+	// SealCommit1 schedules a task to perform the `SealCommit1` stage of a sector being sealed.
+	// If the worker does NOT have the Sealing files for the Sector, it will fetch it from a worker that has them
+	// and store them on one of the Sealing scratch spaces of the sector store before executing the `SealCommit1`.
+	// The call will fail if the Worker is NOT able to allocate space to store the Sealing files.
 	SealCommit1(ctx context.Context, sector storage.SectorRef, ticket abi.SealRandomness, seed abi.InteractiveSealRandomness, pieces []abi.PieceInfo, cids storage.SectorCids) (CallID, error)
+
+	// SealCommit2 schedules a task to perform the `SealCommit2` stage of a sector being sealed.
 	SealCommit2(ctx context.Context, sector storage.SectorRef, c1o storage.Commit1Out) (CallID, error)
+
+	// FinalizeSector schedules a task to perform the `FinalizeSector` stage of a sector being sealed.
+	// As a part of this task, the worker fetches(if required) and stores the cached sealing file on one of the long term
+	// storage directories in the local sector store.
+	// It fetches/keeps around the  Unsealed file in the  Sealing scratch space if `keepUnsealed` is non-zero.
+	// The Unsealed file will be moved to long term storage later when the Miner calls `MoveStorage`.
+	// If `keepUnsealed` is set to zero, it will remove the Unsealed file locally and from all other workers that have it.
 	FinalizeSector(ctx context.Context, sector storage.SectorRef, keepUnsealed []storage.Range) (CallID, error)
+
+	// TODO: ReleaseUnsealed has still not been implemented.
```

Comment:
This is meant to either remove the unsealed file if no ranges to be kept are passed or deallocate parts of the file which we don't need (using `FALLOC_FL_PUNCH_HOLE` / partialfile.Free / https://github.com/filecoin-project/lotus/blob/master/extern/sector-storage/fsutil/dealloc_linux.go#L14)